rllib/agents/ars/ars_tf_policy.py

```diff
@@ -7,11 +7,11 @@
 import ray
 import ray.experimental.tf_utils
 from ray.rllib.agents.es.es_tf_policy import make_session
-from ray.rllib.evaluation.sampler import unbatch_actions
 from ray.rllib.models import ModelCatalog
 from ray.rllib.policy.sample_batch import SampleBatch
 from ray.rllib.utils.filter import get_filter
 from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.utils.space_utils import unbatch
```

Comment:
Moved to `utils/space_utils` to break circular dependency (after using it as well inside policy.py). And renamed `unbatch` (things other than actions can be unpatched as well).